src/components/body/Tabs.js

```diff
@@ -1,51 +0,0 @@
-import { useState } from "react";
-import EduAndSkills from "./bodyparts/EduAndSkills";
-import ExtraCurricular from "./bodyparts/ExtraCurricular";
-import WorkExp from "./bodyparts/WorkExp";
-import "./Tabs.css";
-
-const Tabs = () => {
-  const [toggleTab, setToggleTab] = useState(0);
-
-  const toggleTabHandler = (index) => {
-    setToggleTab(index);
```

Comment:
this function doesnt do anything except call another function.
You could very easier just to `onClick={() => setToggleTab(1)}` below instead